.upgrade.yml

```diff
@@ -0,0 +1,32 @@
+mappings:
```

Comment:
Do you know if this needs to be committed? This file is created when running the upgrader tool and is used for some of the upgrade steps to update references, but I don't think it is required beyond the update process.

Reply:
Deleted!